testsuite/integration/manualmode/src/test/config/arq/arquillian.xml

```diff
@@ -262,6 +262,22 @@
                 <property name="waitForPortsTimeoutInSeconds">8</property>
             </configuration>
         </container>
+
+        <container qualifier="jbossas-sts" default="false" mode="manual">
```

Comment:
I don't really see any reason why this should be a manual mode test. Manual mode tests are not really scalable and add significant amount of time to the test suite, and as such should be reserved for tests that actually require a container to be started and stopped.